server/db/data/seed.js

```diff
@@ -16,27 +16,45 @@ const pool = new Pool({
     console.time('timing seed');
 
     await client.query('BEGIN');
+
+    await client.query(`DROP TABLE IF EXISTS videos;`);
+    await client.query(`DROP TABLE IF EXISTS associatedVideos;`);
+
     await client.query(`
       CREATE TABLE IF NOT EXISTS videos(
         name VARCHAR(50) NOT NULL,
-        url VARCHAR(100) NOT NULL,
-        associatedVideos INT [] NOT NULL
+        url VARCHAR(100) NOT NULL
+        );
+    `);
+
+    await client.query(`
+      CREATE TABLE IF NOT EXISTS associatedVideos(
+        id INT NOT NULL,
+        associatedId INT NOT NULL
         );
     `);
 
     console.log('writing to database!');
 
-    //const path = '/Users/aysun/Documents/hr/video-player-and-carousel/server/db/data/csv/data.csv';
-    const path = '/Users/aysun/Documents/hr/video-player-and-carousel/data.csv';
+    const videosPath = '/Users/aysun/Documents/hr/video-player-and-carousel/server/db/data/csv/VideosData.csv';
+    const associatedPath = '/Users/aysun/Documents/hr/video-player-and-carousel/server/db/data/csv/AssociatedData.csv';
 
```

Comment:
Would it be possible to use relative paths instead of absolute paths? This will be useful for when you deploy!